### ISSUE TITLE
Support Model = Sequential() ? 

### ISSUE THREAD

**Issue by dmracek** [albermax/innvestigate#122]
Hello, I've enjoyed exploring this tool. Thank you.

I'm wondering if it would be possible if iNNvestigate can handle a LSTM, that is, a sequential model. I'm not seeing anything comparable in the utils/tests/networks/base.py. Also, curious if there is a way to stack 2D convolutions and LSTM layers. More specifically, something along the lines of what's below, perhaps using TimeDistributed as a way to extend the sentiment_analysis.ipynb

https://github.com/keras-team/keras/issues/4172

nb_filters=32
kernel_size=(3,3)
pool_size=(2,2)
nb_classes=10
batch_size=64

model=Sequential()

model.add(TimeDistributed(
    Convolution2D(
        nb_filters, kernel_size[0], kernel_size[1], border_mode="valid"), input_shape=[4, 1, 56,14]))
model.add(TimeDistributed(Activation("relu")))
model.add(TimeDistributed(Convolution2D(nb_filters, kernel_size[0], kernel_size[1])))
model.add(TimeDistributed(Activation("relu")))
model.add(TimeDistributed(MaxPooling2D(pool_size=pool_size)))
model.add(TimeDistributed(Flatten()))
model.add(TimeDistributed(Dropout(0.25)))
model.add(LSTM(5))
model.add(Dense(50))
model.add(Dense(nb_classes))
model.add(Activation("softmax"))

**Comment by albermax** [comment]
Hi Derek,

for some methods it should be possible, like input times gradient or IntegratedGradients. Others do not support LSTM layers per definition. But you are right, we did not test it yet.

Would you like to test it and add another sentiment analysis example with LSTMs?
